lib/Auth/Process/COmanageDbClient.php

```diff
@@ -334,15 +290,14 @@ public function __construct($config, $reserved)
         }
     }
 
-    public function process(&$state)
-    {
+    public function process(&$state) {
```

Comment:
Opening braces for methods MUST go on the next line, and closing braces MUST go on the next line after the body. (see https://www.php-fig.org/psr/psr-2/#1-overview)